test/basic.html

```diff
@@ -173,6 +202,45 @@
           assert.isTrue(smallRect.height >= 1 * parseFloat(smallStyle.fontSize));
         });
       });
+
+      suite('ink size', function() {
+        var checkboxes;
+
+        setup(function() {
+          checkboxes = fixture('WithDifferentSizes2');
+        });
+
+        test('`--paper-checkbox-ink-size` sets the ink size', function() {
```

Comment:
I thought the next test checked that the ripple was 8/3, not an exact pixel difference. Basically, how can you tell from the tests this isn't a breaking change?